snap/README.md

```diff
@@ -21,37 +21,70 @@ The snap is published in the snap store at https://snapcraft.io/edgex-app-servic
 You can see the current revisions available for your machine's architecture by running the command:
 
 ```bash
-$ snap info edgex-app-service-configurable
+snap info edgex-app-service-configurable
 ```
 
 The latest stable version of the snap can be installed using:
 
 ```bash
-$ sudo snap install edgex-app-service-configurable
+sudo snap install edgex-app-service-configurable
 ```
 
-The 2.0 (Ireland) release of the snap can be instaing using:
+The 2.1 (Jakarta) release of the snap can be instaing using:
 
 ```bash
-$ sudo snap install edgex-app-service-configurable --channel=2.0
+sudo snap install edgex-app-service-configurable --channel=2.1
 ```
 
 The latest development version of the snap can be installed using:
 
 ```bash
-$ sudo snap install edgex-app-service-configurable --edge
+sudo snap install edgex-app-service-configurable --edge
 ```
 
 **Note** - the snap has only been tested on Ubuntu Core, Desktop, and Server.
 
 ## Using the EdgeX App Service Configurable snap
 
-The App Service Configurable application service allows a variety of use cases to be met by simply providing configuration (vs. writing code). For more information about this service, please refer to the README. As with device-mqtt, this service is disabled when first installed, as a profile must first be selected (see below) before the service is started. As with other EdgeX snaps, the `configuration.toml` files are found in the snap’s writable area:
+The App Service Configurable application service allows a variety of use cases to be met by simply providing configuration (vs. writing code). For more information about this service, please refer to the [README]
+(https://github.com/edgexfoundry/app-service-configurable/blob/main/README.md). As with device-mqtt, this service is disabled when first installed, as a profile must first be selected (see [profile](#profiles) section below) before the service is started. As with other EdgeX snaps, the `configuration.toml` files are found in the snap’s writable area: `/var/snap/edgex-app-service-configurable/current/config/res/`.
```

Comment:
1. Please use a relative link for the README so that the URL is constructed for the version/tag being viewed.
2. [optional] Add new lines between sentences to improve version control. Markdown will put them back together.
3. Comparing to device-mqtt and other snaps is out of context and may become invalid. I understand that this wasn't added by this PR, and I am suggesting to do improve it here since the sentences are being modified anyway.

Reply:
Updated.